src/sentry/utils/sdk.py

```diff
@@ -65,6 +65,7 @@
     "sentry.ingest.transaction_clusterer.tasks.cluster_projects": settings.SENTRY_RELAY_TASK_APM_SAMPLING,
     "sentry.tasks.process_buffer.process_incr": 0.01,
     "sentry.replays.tasks.delete_recording_segments": settings.SAMPLED_DEFAULT_RATE,
+    "sentry.replays.tasks.delete_replay_recording_async": settings.SAMPLED_DEFAULT_RATE,
```

Comment:
```suggestion
    "sentry.replays.tasks.delete_replay_recording": settings.SAMPLED_DEFAULT_RATE,
```
If you use my suggestion above